src/modules/objectsearch/__tests__/ObjectSearchComponent-test.js

```diff
@@ -42,21 +42,15 @@ describe('ObjectSearchComponent', () => {
   it('should display object 1', () => {
     const wrapper = shallow(
       <ObjectSearchComponent
-        location={{
-          query: {
-            page: '1'
-          },
-          pathname: '/#/search/objects'
+        store={{
+          loaded: true,
+          data: testData,
+          params: {
+            currentPage: 1,
```

Comment:
add to params and run test (to update snapshot):
museumNo: '1234',
subNo: '455',
term: 'Lololol',